backend/cmd/headlamp.go

```diff
@@ -675,17 +709,23 @@ func refreshAndCacheNewToken(oidcAuthConfig *kubeconfig.OidcConfig,
 	// get provider
 	provider, err := oidc.NewProvider(context.Background(), oidcAuthConfig.IdpIssuerURL)
 	if err != nil {
+		utils.Log(utils.LevelError, nil, err, "Error: failed to get provider")
+
 		return "", err
 	}
 
 	// get refresh token from cache
 	refreshToken, err := cache.Get(context.Background(), fmt.Sprintf("oidc-token-%s", token))
 	if err != nil || refreshToken == "" {
+		utils.Log(utils.LevelError, nil, err, "Error: failed to get refresh token")
+
 		return "", err
 	}
 
 	rToken, ok := refreshToken.(string)
 	if !ok {
+		utils.Log(utils.LevelError, nil, err, "Error: failed to cast refresh token")
```

Comment:
```suggestion
		utils.Log(utils.LevelError, nil, err, "failed to cast refresh token")
```

since it is passed with Error as severity I think it is not necessary to add Error to the message.